### ISSUE TITLE
About window broken in Electron 5

### ISSUE THREAD

**Issue by dtslvr** [rhysd/electron-about-window#37]
I have upgraded to _Electron 5.0.1_ and located that the _about window_ is broken. There is no text anymore, just the (broken) icon with a "App icon" label".

**Comment by Vages** [comment]
Same here. Here's how it looks.

<img width="512" alt="image" src="https://user-images.githubusercontent.com/3520744/57367249-f025a280-7188-11e9-8e79-10fea39e8448.png">

**Comment by Vages** [comment]
I found a fix. Will be posting a PR in a few minutes.

**Comment by Vages** [comment]
While the PR is waiting to be merged, you can use [`patch-package`](https://www.npmjs.com/package/patch-package) and add the following file to your project's `patches` folder. Name the file `about-window+1.12.1.patch`:

```
diff --git a/node_modules/about-window/src/index.js b/node_modules/about-window/src/index.js
index 6a3cb6f..7bc8545 100644
--- a/node_modules/about-window/src/index.js
+++ b/node_modules/about-window/src/index.js
@@ -100,6 +100,11 @@ function openAboutWindow(info_or_img_path) {
         titleBarStyle: 'hidden-inset',
         show: !info.adjust_window_size,
         icon: info.icon_path,
+        webPreferences: {
+            // For security reasons, nodeIntegration is no longer true by default when using Electron v5 or later
+            // nodeIntegration can be safely enabled as long as the window source is not remote
+            nodeIntegration: true,
+        },
     }, info.win_options || {});
     window = new (electron_1.BrowserWindow || electron_1.remote.BrowserWindow)(options);
     window.once('closed', () => {
```

**Comment by 4rno** [comment]
Thanks @Vages ! On my side, I'm gonna use your fork repo while we wait your PR to be merged.
`"about-window": "Vages/electron-about-window#master",`

**Comment by 4rno** [comment]
Actually, you can also use the `win_options` property when you call the `openAboutWindow()` function, and set nodeIntegration to true.

```
openAboutWindow({
    ...
    win_options: {
        webPreferences: {
            nodeIntegration: true
        }
    }
})
```

`win_options | Options of 'About This App' window. It is merged into default options. Optional | BrowserWindow options object`

**Comment by rhysd** [comment]
This issue was fixed by v1.13.0. Please update package.

**Comment by dtslvr** [comment]
> This issue was fixed by v1.13.0. Please update package.

Thanks a lot, @rhysd and @Vages!
